src/pages/components/breadcrumb/style.mdx

```diff
@@ -6,82 +6,194 @@ description:
 tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 ---
 
+<PageDescription>
+
+The following page documents visual specifications such as color, typography,
+structure, and size.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Color</AnchorLink>
+<AnchorLink>Typography</AnchorLink>
+<AnchorLink>Structure</AnchorLink>
+<AnchorLink>Size</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
 ## Color
 
-| Element      | Property | Color token           |
-| ------------ | -------- | --------------------- |
-| Link         | color    | `$link-primary`       |
-| Link:hover   | color    | `$link-primary-hover` |
-| Link:active  | color    | `$text-primary`       |
-| Link:focus   | outline  | `$focus`              |
-| Slash        | color    | `$text-primary`       |
-| Current page | color    | `$text-primary`       |
+| Element        | Property   | Color token           |
+| ------------   | --------   | --------------------- |
+| Text: enabled  | text-color | `$link-primary`       |
+| Text: current  | text-color | `$text-primary`       |
+| Slash          | text-color | `$text-primary`       |
+| Overflow: icon | svg        | `$link-primary`       |
+
+<Row>
+<Column colLg={8}>
+
+![Breadcrumb color](images/breadcrumb-style-color-1.png)
+
+<Caption>Breadcrumb color.</Caption>
+
+</Column>
+</Row>
 
 <Row>
 <Column colLg={8}>
 
-![Breadcrumb states and color](images/breadcrumb-style-color.png)
+![Breadcrumb overflow color](images/breadcrumb-style-color-overflow-2.png)
+
+<Caption>Breadcrumb overflow color.</Caption>
+
+</Column>
+</Row>
+
+### Breadcrumb interactive state color
+
+| State      | Element            | Property      | Color token          |
+| -------    | ------------------ | ------------- | ------------------   |
+| Hover      | Text               | text-color    | `$link-primary-hover`|
+| Focus      | Text               | text-color    | `$link-primary`      |
+|            | Border             | border        | `$focus`             |
+| Active     | Text               | text-color    | `$text-primary`      |
+|            | Border             | border        | `$focus`             |
+| Current    | Text               | text-color    | `$text-primary`      |
+
+<Row>
+<Column colLg={8}>
+
+![Breadcrumb interactive state color](images/breadcrumb-style-interactive-state-color-1.png)
+
+<Caption>Breadcrumb interactive state color.</Caption>
+
+</Column>
+</Row>
+
+### Breadcrumb overflow interactive state color
+
+A group of breadcrumbs with overflow content uses the overflow menu component to disclose additional breadcrumbs in a menu view. A tooltip component appears in the hover, focus, and active states. Use the menu [menu](/components/menu/style) and tooltip [tooltip](/components/tooltip/style) colors for these instances.
```

Comment:
```suggestion
A group of breadcrumbs with overflow content uses the overflow menu component to disclose additional breadcrumbs in a menu view. A tooltip component appears in the hover, focus, and active states. Use the [menu](/components/menu/style) and [tooltip](/components/tooltip/style) colors for these instances.
```

Reply:
Above^ If you press the "Commit suggestion" button dropdown, and then select "Commit changes", it will automatically make the changes for you.